model_analyzer/config/generate/search_parameters.py

```diff
@@ -134,7 +134,9 @@ def _populate_parameters(self) -> None:
         self._populate_batch_sizes()
 
         if not self._is_composing_model:
-            if self._config.is_request_rate_specified(self._parameters):
+            if self._parameters and self._config.is_request_rate_specified(
```

Comment:
The pre-condition of self._parameters is not correct and should be removed. There are other cases (based on just values in self._config) that can cause this method to return true.